### PR TITLE
Feat/enhanced video component

### PR DESCRIPTION
**Pull request by brunadafonseca**

### CODE REVIEW

File: dato.config.js [hunk 1]
```diff
@@ -68,6 +68,11 @@ function transformItem(item) {
     $('img').remove()
     item.body = $('body').html()
   }
+  else if (item.type === 'video') {
+    if (item.video && item.title) {
+      item.video.title = item.title
```
Comment (jurgenbelien) [inline dato.config.js:73]: I wouldn't put the `title` on the `item.video`. The `title` is the title of the `item`. The video contains all things related to the _external_ video source.

File: dato.config.js [hunk 2]
```diff
@@ -68,6 +68,11 @@ function transformItem(item) {
...
+    }
```
Comment (jurgenbelien) [inline dato.config.js:74]: It also needs the values for autoplay, loop and mute from Dato. As it stands now, the toggles, they do nothing 

https://www.google.nl/search?q=the+goggles+they+do+nothing

File: src/client/components/responsive-video/responsive-video.stories.js [hunk 3]
```diff
@@ -17,14 +18,85 @@ storiesOf('Responsive video', module)
...
+        video: {
+          "url": "https://www.youtube.com/watch?v=ZPIIT6siIaE&list=PL8HcoiB7g3NlaU-UU1boptT4tkz3KOXT2",
+          "title": "Een surfer neemt het op tegen de plastic vervuiling",
+          "width": 480,
+          "height": 270,
+          "provider": "youtube",
+          "providerUid": "ZPIIT6siIaE",
+          "thumbnailUrl": "https://i.ytimg.com/vi/ZPIIT6siIaE/hqdefault.jpg",
+          "autoplay": false,
+          "loop": false,
```
Comment (jurgenbelien) [inline src/client/components/responsive-video/responsive-video.stories.js]: Please define this the way it should come from dato, ie do not nest loop, autoplay, mute and title in the video property.

**Review by jurgenbelien — CHANGES_REQUESTED** [review]
It doesn't work yet with content from dato. Also make sure that the nesting of those props (autoplay, loop, mute, title) are always the same, as not to confuse anyone

File: dato.config.js (hunk 2 above)
Comment (brunadafonseca) [reply dato.config.js:74]: In dato we just overwrite the video title if there is one.

**Review by jurgenbelien — APPROVED** [review]

**Pull request merged**